yggdrasilwasm/src/lib.rs

```diff
@@ -8,6 +8,17 @@ use wasm_bindgen::prelude::*;
 
 type CustomStrategyResults = HashMap<String, bool>;
 
+const KNOWN_STRATEGIES: [&str; 8] = [
```

Comment:
This looks like a copy of the YggdrasilFFI version. Can we just move both of these declarations into Yggdrasil itself and then they can both just reference that?

Reply:
Yep, good point! I assume something like this will just work: https://github.com/Unleash/yggdrasil/pull/226/commits/fceb36e422a89b3ecf3c9815430b6bdd543fd9e3